src/client/components/UserEditor.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { User } from '../../shared/User';
+import { remult } from '../utils/api-facade';
+import { Box } from '@mui/material';
+import { ErrorInfo } from 'remult';
+
+const userRepo = remult.repo(User);
+
+interface IProps {
```

Comment:
The `IThing` convention isn't a common one and is actually officially recommended against by TS. I simply name "FooProps" for "Foo" component.

Reply:
OK - so it should be in this case UserEditorProps?

I took the IProps from the other components in this project ,trying to be consistent to them

Reply:
Yes, I'd name it "UserEditorProps". As for conventions:
1. IThing is a convention in some codebases, so you're obviously free to adopt it. But it is way less common than "just name things" without the Hungarian notation.
2. I like consistency too, but if a project/configuration doesn't suit my own idea then I decide differently and establish my own convention. In other words, the value of a convention less about being consistent and more about what value does following it brings to the maintainer. When I see no or little value, I come up with something better (hopefully 😅 ).

Reply:
:)